packages/storybook/.storybook/blocks/banner.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
```

Comment:
I would prefer not to use React at all in the storybook, you can create your own block without it, see: https://storybook.js.org/docs/api/doc-block-useof (carefull to check `Web Components` on the top selector)